manifests/Bytedance/Feishu/3.22.5.yaml

```
@@ -0,0 +1,15 @@
+Id: Bytedance.Feishu
+Name: Bytedance Feishu
+Version: 3.22.5
+Publisher: Beijing Feishu Technology
+Homepage: https://www.feishu.cn/
+License: Copyright © 2020 Beijing Feishu Technology Co., Ltd. All Rights Reserved.
+Tags: Feishu
+InstallerType: exe
+Installers:
+    Arch: x86
+    Url: https://sf3-eecdn-tos.pstatp.com/obj/ee-appcenter/Feishu-3.22.5.exe
+    Sha256: 8e58059149e55cb758f410a9808163e1179cdf77ea1e70021849f00b1427d62e
+Switches:
+    Silent: /S
+    SilentWithProgress: /S
```

```suggestion
    Switches:
        Silent: /S
        SilentWithProgress: /S
```
Add indents for the `Switches` field may help your commit to pass the validation test.